sdk/maps/Azure.Maps.Geolocation/tests/Samples/GeolocationClientSamples.cs

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+#region Snippet:GeolocationImportNamespace
+using Azure.Maps.Geolocation;
+#endregion
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.Maps.Geolocation.Tests
+{
+    public class GeolocationClientSamples : SamplesBase<GeolocationClientTestEnvironment>
+    {
+        public void GeolocationClientViaAAD()
+        {
+            #region Snippet:InstantiateGeolocationClientViaAAD
+            // Create a MapsGeolocationClient that will authenticate through Active Directory
+            var credential = new DefaultAzureCredential();
+            var clientId = "<My Map Account Client Id>";
```

Comment:
Based on comments from Route and Render. The same pattern is recommended for other snippets to avoid referencing test constructs in samples.
```suggestion
#if SNIPPET
            TokenCredential credential = new DefaultAzureCredential();
            string clientId = "<Your Map ClientId>";
#else
            TokenCredential credential = TestEnvironment.Credential;
            string clientId = TestEnvironment.MapAccountClientId;
#endif
```